src/Blog/Tag/TagRepository.php

```diff
@@ -17,11 +17,8 @@
 
 final class TagRepository extends Repository
 {
-    private ORMInterface $orm;
-
-    public function __construct(Select $select, ORMInterface $orm)
+    public function __construct(private ORMInterface $orm, Select $select,)
```

Comment:
Let's keep the same order unless calls are refactored too. Also redundant comma at the end.